server/resp/src/main/java/org/infinispan/server/resp/RequestHandler.java

```diff
@@ -0,0 +1,39 @@
+package org.infinispan.server.resp;
+
+import java.util.List;
+
+import org.infinispan.Cache;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufAllocator;
+import io.netty.buffer.ByteBufUtil;
+import io.netty.channel.ChannelHandlerContext;
+
+public interface RequestHandler {
+   default RequestHandler handleRequest(ChannelHandlerContext ctx, Cache<byte[], byte[]> cache, String type, List<byte[]> arguments) {
+      ctx.writeAndFlush(stringToByteBuf("-ERR unknown command" + "\r\n", ctx.alloc()));
```

Comment:
```suggestion
      ctx.writeAndFlush(stringToByteBuf("-ERR unknown command\r\n", ctx.alloc()));
```
Any reason to concatenate strings?